ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/events/AlterTableEvent.java

```diff
@@ -116,4 +157,13 @@ private String buildCommandString(String cmdStr, Table tbl) {
     }
     return ret;
   }
+
+  private static String getTablePropsForCustomStorageHandler(Map<String, String> tableProperties){
```

Comment:
should we consolidate these 2 implementations of this method into some utility class?